doc/references.bib

```diff
@@ -141,6 +141,15 @@ @article{BergScherg1994
   year = {1994}
 }
 
+@inproceedings{BigdelyShamloEtAl2013,
+  author={Bigdely-Shamlo, Nima and Kreutz-Delgado, Kenneth and Robbins, Kay and Miyakoshi, Makoto and Westerfield, Marissa and Bel-Bahar, Tarik and Kothe, Christian and Hsi, Jessica and Makeig, Scott},
```

Comment:
adding the DOI here.

Also note that the remaining references have a space between `author`, the `=`, and the content like `{....}`, whereas you do: `author={....}` So a nitpicky reviewer may ask you to be consistent :-)



```suggestion
  author={Bigdely-Shamlo, Nima and Kreutz-Delgado, Kenneth and Robbins, Kay and Miyakoshi, Makoto and Westerfield, Marissa and Bel-Bahar, Tarik and Kothe, Christian and Hsi, Jessica and Makeig, Scott},
  doi={10.1109/GlobalSIP.2013.6736796},
```